### PR TITLE
add project finc of Leipzig University Library

### PR DESCRIPTION
**Pull request by ndege**

### CODE REVIEW

**Comment by stweil** [comment]
@ndege, what about adding your GitHub account to `list.md`, too?

File: list.md [hunk 1]
```diff
@@ -85,6 +85,9 @@ Deutsche Zentralbibliothek für Medizin (ZB MED)
 UB Ilmenau
 [https://github.com/ubilmenau](https://github.com/ubilmenau)
 
+project finc
```
Comment (stweil) [inline list.md]: I suggest to be more precise, e. g. by using "Project finc (UB Leipzig)".

Comment (ndege) [inline list.md]: Hi Stefan,

Ok , I agree. I try to find some more precise. The project is based mainly at Leipzig University Library, but the focus isn't so restricted to one institution. So I'll reflect about a better naming for a while and send a new suggestion.

Comment (stweil) [inline list.md]: @hbunke and all other contributors: maybe a new heading for projects with an own GitHub account could be added to `list.md`, something like `Bibliothekarische Projekte`. Then projects like `finc` could be added there. Ideally such projects are not restricted to one institution, but IMHO it is fine to name the institution where a project was started or is mainly based. `https://github.com/goobi/` is another example which would fit under the new heading.

**Comment by hbunke** [comment]
Adding projects is, of course, a good idea. For now I'd tend to simply add them to the 'Libraries and other Institutions' section (I've added the approbiate headline).

**Comment by ndege** [comment]
Thanks for merging!

**Pull request merged**